BP/localization/english/religion/bp_religion_paganism_l_english.yml

```diff
@@ -856,10 +887,11 @@ l_english:
  gurzil_high_god_name:0 "Gurzil"
  gurzil_high_god_name_possessive:0 "Gurzil's"
  #Berber
- berber_pagan:0 "Amazighism"
- berber_pagan_adj:0 "Amazighism"
- berber_pagan_adherent:0 "Amazighist"
- berber_pagan_adherent_plural:0 "Amazighists"
+ #Achaman is the suprme god in Guanche religion, so used to refer to all berber_pagans
```

Comment:
Aren't the Guanche a very small and minor group of isolated Berbers ?

Reply:
They are, but its the only purely Berber faith where I can find pantheon references. Not the best reference, but I was deeply unhappy with Amazighism (since it's a cultural endonym, not a religious one). I can rename it Amazighism for lack of a better endonym.

Maybe split off Guanche's faith from Berber_Pagan as it had some independent practices (mummification, for example)?

Reply:
In my opinion it is too much of a stretch, the old name is better. As for splitting off the Guanche, I don't know enough to have a real opinion, the question might rather be if it is that important and worth the time for only one peripheral province.

Reply:
Amazighism changed.
As for Guanche, it's three counties now, but I get what you mean.